src/providers/DataTypes.ts

```diff
@@ -13,3 +13,12 @@ export type ProvidedOrganizationData = {
   organizations?: ServerOrganization[];
   reloadData: () => void;
 };
+
+export type TimeZoneDescription = {
```

Comment:
Should we add this type to a file in src/types?

Reply:
that would make sense as it is not specifically tied to the Provider